includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -350,7 +332,7 @@ public function get_product_data() {
 		$data                    = [];
 		$data['total']           = [
 			'label'   => apply_filters( 'wc_stripe_payment_request_total_label', $this->total_label ),
-			'amount'  => WC_Stripe_Helper::get_stripe_amount( $this->get_product_price( $product ) ),
+			'amount'  => WC_Stripe_Helper::get_stripe_amount( $product->get_price() ),
```

Comment:
I'm removing `get_product_price` since is not relevant anymore. This amount will never be used in the frontend. Its purpose is just to be able to initialize the PR button when page loads.